string YamlFileName
string YamlCalibrationFileName
string CADName
string MarkerName
string SLS2ServerName
string SLS2IpAddress
float64 VoxelGridLeafSize
---
bool ReturnStatus
string ReturnMessage
string NumerizedMeshName
